src/workers/postUpdated.ts

```diff
@@ -301,7 +301,6 @@ const updatePost = async ({
 
   data.id = databasePost.id;
   data.title = title;
-  data.visible = updateBecameVisible;
```

Comment:
But deleting this wont actually set community picks to visible so not sure this is right?

Reply:
Can you specify exact case where it would break?

Reply:
Just from my head:
User adds community pick
Think we add as not visible?
Then yggdrasil send update
Becomes visible

Reply:
`should update the post and make it visible if title is available` test confirms that case works. `p1` is `visible: false`, then update with title is sent, post becomes visible.

Reply:
Ah ok if thats the case then its fine, but then what did this evaluation do?
Do we set it somewhere else as well?

Reply:
I think the right fix is actually changing the condition of updateBecameVisible. The bug there is that we flip the visibility but actually what we need is to define it as true if title is defined. Like Chris I'm not sure in the current implementation how it ever becomes true 🥵

Reply:
@rebelchris we set it in `fixData`, that is where `data.visible` comes from and it is set.

Reply:
We can continue at https://github.com/dailydotdev/daily-api/pull/1608#discussion_r1427751683